<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettingsWidget</class>
 <widget class="QWidget" name="GeneralSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
   <item row="7" column="1">
    <widget class="QCheckBox" name="updates">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Check for updates automatically</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="launch">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Launch application with media players</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="sendCrashReports">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Send crash reports to Last.fm</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="showAs">
     <property name="text">
      <string>Show application icon in menu bar</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="beta">
     <property name="text">
      <string>Update to beta versions - Warning: only for the brave!</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="showDock">
     <property name="text">
      <string>Show dock icon</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="mediaKeys">
     <property name="text">
      <string>Enable media keys</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="notifications">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Show desktop notifications</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="languages">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="languageLabel">
     <property name="text">
      <string>Language:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
